rero_ils/modules/documents/mappings/v6/documents/document-v0.0.1.json

```diff
@@ -53,6 +53,23 @@
             },
             "qualifier": {
               "type": "keyword"
+            },
+            "mefidentifiers": {
```

Comment:
I'm not sure for this too. After the $ref resolution, what do we have in the document JSON? These identifiers?

Reply:
the jsonresolver_mef_person.py reduces the $ref to this